pkg/backup/restore_job.go

```diff
@@ -85,6 +85,10 @@ import (
 // tables we process in a single txn when restoring their table statistics.
 const restoreStatsInsertBatchSize = 10
 
+// restoreRetryPolicySwitchThreshold is the fraction of the job that must
+// be completed before we switch to the secondary retry policy for RESTORE.
+const restoreRetryPolicySwitchThreshold = 0.3
```

Comment:
This is a pretty arbitrarily chosen limit, so if anyone has a rationalization for a specific threshold, please let me know!